Src/WitsmlExplorer.Api/Extensions/HttpContextExtensions.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+
+using Microsoft.AspNetCore.Http;
+
+using WitsmlExplorer.Api.HttpHandlers;
+namespace WitsmlExplorer.Api.Extensions
+{
+    public static class HttpContextExtensions
+    {
+        public static string GetOrCreateWitsmlExplorerCookie(this HttpContext httpContext, string environment)
```

Comment:
Why is the cookie dependent on the environment?

Reply:
Thanks I will try to check if we can make these changes.
The reason for including environment in the cookie name was to better distinguish and not overwrite the cookie if testing different environments. Having the same name might not be a big issue though